src/httpserver.rs

```diff
@@ -63,8 +63,8 @@ async fn handle(req: Request<Body>, snare: Arc<Snare>) -> Result<Response<Body>,
         return Ok(res);
     }
 
-    // We now check thatthe per-repo program to run while making sure that we aren't tricked into
-    // executing a file outside of the repos dir.
+    // We now check that there is per-repo program for this repository and that we haven't been
```

Comment:
We now check that there is [a] per-repo program

Reply:
Fixed in https://github.com/softdevteam/snare/pull/14/commits/30b0799f8c96cfcb6cc031ed0788564f6d0bcf21.